include/net/net_mgmt.h

```diff
@@ -149,6 +149,9 @@ struct net_mgmt_event_callback {
 		 * be called in case those events come. Such mask can be
 		 * modified whenever necessary by the owner, and thus will
 		 * affect the handler being called or not.
+		 * Note that only the command part is treated as a mask,
+		 * matching one to several commands. Layer and layer code will
```

Comment:
I suggest leaving this addition out as an implementation detail on how event set filtering works